editor/plugins/path_3d_editor_plugin.cpp

```diff
@@ -40,17 +40,28 @@
 #include "scene/gui/menu_button.h"
 #include "scene/resources/curve.h"
 
-static bool _is_in_handle(int p_id, int p_num_points) {
-	int t = (p_id + 1) % 2;
-	int idx = (p_id + 1) / 2;
-	// order of points is [out_0, out_1, in_1, out_2, in_2, ... out_n-1, in_n-1, in_n]
-	if (idx == 0) {
-		return false;
-	} else if (idx == (p_num_points - 1)) {
-		return true;
-	} else {
-		return (t == 1);
-	}
+// Map handle id to control point id and handle type.
+enum HandleType {
+	IN = 0,
+	OUT,
+	TILT,
+};
+
+struct HandleInfo {
+	int point_idx; // Index of control point
```

Comment:
```suggestion
	int point_idx; // Index of control point.
```
[Comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide) nitpick